src/AppInstallerCommonCore/Manifest/ManifestYamlPopulator.cpp

```diff
@@ -137,20 +137,52 @@ namespace AppInstaller::Manifest
         // Common fields across versions
         std::vector<FieldProcessInfo> result =
         {
-            { "ManifestVersion", [](const YAML::Node&)->ValidationErrors { /* ManifestVersion already populated. Field listed here for duplicate and PascalCase check */ return {}; } },
-            { "Installers", [this](const YAML::Node& value)->ValidationErrors { m_p_installersNode = &value; return {}; } },
-            { "Localization", [this](const YAML::Node& value)->ValidationErrors { m_p_localizationsNode = &value; return {}; } },
-            { "Channel", [this](const YAML::Node& value)->ValidationErrors { m_p_manifest->Channel = Utility::Trim(value.as<std::string>()); return {}; } },
+            { "ManifestVersion", [](const YAML::Node&, std::any&)->ValidationErrors { /* ManifestVersion already populated. Field listed here for duplicate and PascalCase check */ return {}; } },
+            { "Installers", [this](const YAML::Node& value, std::any&)->ValidationErrors { m_p_installersNode = &value; return {}; } },
+            { "Localization", [this](const YAML::Node& value, std::any&)->ValidationErrors { m_p_localizationsNode = &value; return {}; } },
+            {
+                "Channel",
+                [](const YAML::Node& value, std::any& any)->ValidationErrors
+                {
+                    Manifest* manifest = std::any_cast<Manifest*>(any);

```

Comment:
I think I would have gone with something to make this easier to remain one line, like:
```
template <typename Ptr>
Ptr* any_ptr(std::any& any) { return std::any_cast<Ptr*>(any); }
...
any_ptr<Manifest>(any)->Channel = ...;
...
```
Which is mostly in the not creating a local, but also is slightly shorter.  If you wanted to just not create a local it would be almost as short, but the function makes it easier to add something if needed.